pkg/trait/jolokia.go

```diff
@@ -124,7 +114,7 @@ func (t *jolokiaTrait) Apply(e *Environment) error {
 
 	jolokiaFilepath := ""
 	for _, ar := range e.IntegrationKit.Status.Artifacts {
-		if strings.HasPrefix(ar.ID, "org.jolokia.jolokia-jvm") {
+		if strings.HasPrefix(ar.ID, "org.jolokia.jolokia-agent-jvm") {
```

Comment:
We should not change this until the new catalog is ready. Let's make it compatible with the catalog we are using and we'll change to the new values once we have the new catalog.

Reply:
The old and new artifact are expected.